pkgs/development/php-packages/composer/default.nix

```diff
@@ -40,7 +40,7 @@ php.buildComposerProject (finalAttrs: {
       --prefix PATH : ${lib.makeBinPath [ _7zz cacert curl git unzip xz ]}
   '';
 
-  vendorHash = "sha256-dNNV9fTyGyRoGeDV/vBjn0aMgkaUMsrKQv5AOoiYokQ=";
+  vendorHash = "sha256-4s912b04PTPbK9+suni9uJ2Cx4qDb8CkMLy6y25a1eQ=";
```

Comment:
Yeah, this is bad. If we want to not break BC for outside consumers, we would need to introduce `mkComposerRepository2` and `buildComposerProject2`, and deprecate the old one.